src/content/introduction/new.md

```diff
@@ -0,0 +1,100 @@
+---
+id: 1
+title: New Features
+category: introduction
+slug: new
+type: docs
+scope:
+  - range
+  - sampleData
+---
+
+# New Features
+
+An intro about new features! A link to the changelog
+
+## Backgrounds for VictoryChart
+
+VictoryChart now includes an optional [`backgroundComponent`](/docs/victory-chart#backgroundcomponent) prop. It will also render the new [`Background`](/docs/victory-primitives#backgroundcomponent) simple component as the default backgroundComponent when the "background" property is included in the VictoryChart style prop.
```

Comment:
I would love to see the copy on this page be a little more conversational. I can help out here if you'd like.